app/check.php

```diff
@@ -21,7 +21,7 @@
 
 $messages = array();
 foreach ($symfonyRequirements->getRequirements() as $req) {
-    /** @var $req Requirement */
+    /* @var $req Requirement */
```

Comment:
@cordoval this change is actually wrong. There is a difference between these two docblocks. The first one actually tokenises to something different to this one. This one is considered a comment, so all annotations have no meaning, where as, before it was considered a phpdoc comment, so annotations have meaning.